examples/burial-form/BurialAllowance.jsx

```diff
@@ -32,16 +32,6 @@ export default function BurialAllowance(props) {
     return allowanceTypes;
   }
 
-  useEffect(() => {
```

Comment:
always love when a refactor leads to the removing code like this. Nice work @harshil1793!